src/plone/restapi/services/auth/login.py

```diff
@@ -73,6 +75,8 @@ def _find_userfolder(self, userid):
             uf = getToolByName(uf_parent, 'acl_users')
             if uf:
                 info = uf._verifyUser(uf.plugins, login=userid)
+            if IPloneSiteRoot.providedBy(uf_parent):
```

Comment:
@jaroel I guess uf_parent means user_folder_parents. More people should read books like "Clean Code" and try to use variable naming that can be understood when reading them (not your fault here of course). Is that check really sufficient? What if we do not have user folders enabled? I am wondering if there is a more robust way of checking for a Plone user. I guess permission checks won't work because the Zope user has them all, right?